examples/dynamo/eg3/solver_mod.x90

```diff
@@ -1,3 +1,4 @@
+! Modifications copyright (c) 2017, Science and Technology Facilities Council
```

Comment:
We should take this opportunity to put in the full 3-clause BSD license here and remove the old copyright notice.

Reply:
Replaced old copyright notice with full BSD license.

Reply:
Corrected BSD license as agreed.